.github/CODEOWNERS

```diff
@@ -4,8 +4,8 @@
 *                   @akirillov @VladimirKravtsov @rpalaznik @farhan5900 @samvantran @alembiewski
 /dashboards         @VladimirKravtsov @akirillov
 /images             @alembiewski @akirillov
-/kudo-operator      @farhan5900 @VladimirKravtsov @akirillov
-/kudo-operator/docs @samvantran @akirillov
+/operators/repository/spark      @farhan5900 @VladimirKravtsov @akirillov
```

Comment:
I think, no need to keep here code owners of nested dirs of the mounted submodule. It's just no sense because if we will like to create a PR with changes in  /operators/repository/spark/** then we will do it in the other repo.

Reply:
this makes sense, fixed, thanks